### DISCUSSION TITLE
Query Regarding Using nnU-Net for Continuous Labels

### DISCUSSION THREAD

**Discussion by mehnaz1985** [MIC-DKFZ/nnUNet#1747] [Q&A]
Please any one can suggest me whether nnUNet can run on continuous ground truth labels? like the ground truth image will contain range of values or continuous values rather integer like other datasets we saw earlier.

**Comment by DarylWM** [comment]
Continuous ground truth labels is a regression problem. Discrete ground truth labels is a segmentation problem, and that's what nnU-Net is good at. I'd suggest looking at a standard U-Net with a sigmoid activation function in the final layer.

**Reply by mehnaz1985** [reply]
Thank you for the reply. Yes, this is more like a regression problem. Can I adapt the nnunet for that, like changing the final layer? Is that too complex? Or should I go for other frameworks?
